npg_qc_viewer/root/src/ui_lanes/lane.tt2

```diff
@@ -51,35 +51,50 @@
 
 [%- FOREACH check_name IN checks_list -%]
   [% IF check_name.match('genotype') && !check_name.match('W30467') %]
-        [% NEXT %]
+    [% NEXT %]
   [% END %]
 [%-
     result = '';
     td_class = 'check_summary';
+
     IF lane_collection && lane_collection.size;
       result_collection = lane_collection.slice('check_name', check_name);
       IF result_collection.defined && result_collection.size;
-         result = result_collection.get(0);
-         IF result.pass == 1;
-           td_class = td_class _ ' passed';
-         ELSIF result.pass == 0;
-           td_class = td_class _ ' failed';
-         ELSE;
-           td_class = td_class _ ' outcome_unknown';
-         END;
-    END;
--%]
+        result = result_collection.get(0);
+        IF result.pass == 1;
+          td_class = td_class _ ' passed';
+        ELSIF result.pass == 0;
+          td_class = td_class _ ' failed';
+        ELSE;
+          td_class = td_class _ ' outcome_unknown';
+        END;
+      ELSIF check_name.match('tag metrics') && tag_index.defined;
```

Comment:
We discussed that, to keep the code flow clear, this code should be in a separate BLOCK preferably callable within TRY-CATCH block on lines 90-93.